README.md

```diff
@@ -557,10 +555,10 @@ Staking scripts allow arbitrary logic for the BorrowerIDs such as a multisig.
 ## Future Directions and Considerations
 
 #### Term Extensions/Renegotiations
-A borrower may renegotiate an active loan with their lender, without closing or defaulting on the loan. This may be to "refinance" the loan, to negotiate a loan term extension, or for whatever other reasons the two parties may agree upon. All such actions would be queryable by prospective lenders in the future, giving them further insight into the nature/creditworthiness of the borrower.
+A borrower may renegotiate an active loan with their lender, without closing or defaulting on the loan. This may be to "refinance" the loan, to negotiate a loan term extension, or for whatever other reasons the two parties may agree upon. All such actions would be queryable by prospective lenders in the future, giving them further insight into the borrower's nature/creditworthiness.
```

Comment:
This is me thinking out loud about how I could possibly implement this...

Add a new field to the `ActiveDatum` which is a map from term to desired value (eg, interest -> 5%, and expiration -> slot x). The borrower can add/remove/change what is in this map. When the Key NFT holder (lender) checks on the loan status, these requests will be visible. The lender can update the loan term if desired **but only to what is in the map**. This constraint would prevent the lender arbitrarily changing loan terms. When the lender updates the loan terms, the map in the datum is reset to an empty map to free up space in the datum.

I'm not sure how much computation this would require.

Reply:
putting renegotiations on backburner for now